client/rest/test/plugins/rosetta/accountRoutes_spec.js

```diff
@@ -188,6 +225,11 @@ describe('account routes', () => {
 	// region coins
 
 	describe('coins', () => {
+		const OTHER_SIGNER_PUBLIC_KEY = '527068DA90B142D98D27FF9BA2103A54230E3C8FAC8529E804123D986CACDCC9';
+		const OTHER_RECIPIENT_ADDRESS = 'TDI2ZPA7U72GHU2ZDP4C4J6T5YMFSLWEW4OZQKI';
+		const TARGET_SIGNER_PUBLIC_KEY = 'ED7FE5166BDC65D065667630B96362B3E57AFCA2B557B57E02022631C8C8F1A6';
+		const TARGET_RECIPIENT_ADDRESS = 'TARZARAKDFNYFVFANAIAHCYUADHHZWT2WP2I7GI';
```

Comment:
do you intend to have OTHER* and TARGET* ids to not point to same account?

Reply:
i mean this reads like it is sending to self (even though it is not)
```
const transaction = createTransactionJson(mosaicsToReceive, OTHER_SIGNER_PUBLIC_KEY, OTHER_RECIPIENT_ADDRESS);
```

Reply:
Ok. will update hopefully it will be better 😅